### ISSUE TITLE
[Bug]: GrechTech Multiblocks Overclocking not properly working

### ISSUE THREAD

**Issue by TheLastPaladin** [AllTheMods/ATM-9#1578]
### Possible Fixes

Yes

### Modpack Version

0.2.56

### What happened?

I've updated my server today and all GregTech Multiblocks have been downgraded in their power tier. I've noticed that overclocking still behaves as intended if multiple energy hatches are used. However 1A, 4A and 16A Energy Input Hatches are now effectively the same. This results in the (significantly more expensive) 4A and 16A hatches being basically pointless.
I'm assuming this is a bug since adding this behaviour would be illogical and contradictory to how GT energy works.

Sadly I could not find out if this is an issue direcly causes by GregTech itself, in that case I will move this report elsewhere.

**Comment by screret** [comment]
intended. 4A hatches no longer overclock multiblocks, use 2 hatches.

**Comment by screret** [comment]
no tier skipping for you

**Comment by TheLastPaladin** [comment]
Tier skipping was never possible because of adjusted crafting recipes. Also I can still overclock my machines since two 1A Hatches achieve an overclocked power tier

**Comment by Kevin-Marsh** [comment]
This is intended by the GTCEU developers.

**Comment by screret** [comment]
> Tier skipping was never possible because of adjusted crafting recipes. Also I can still overclock my machines since two 1A Hatches achieve an overclocked power tier. If I were to e.g. change all my 4A hatches to 2x1A hatches nothing would change.

that's what we wanted to achieve, don't worry.

**Comment by screret** [comment]
what's "most multiblocks" that only applies to large assembler & circuit assembler

**Comment by TheLastPaladin** [comment]
Alloy Blast Smelter doesn't accept more than 1 energy hatch either. I'm testing which MBs are affected rn

**Comment by TheLastPaladin** [comment]
This affects (only speaking if multiblocks here) circuit assembler and assembler (as you mentioned intended) and also the Alloy Blast Furnace, Laser Engraver and possibly more. Since I have to construct the multiblocks in order to test if they are affected those are the only ones at the moment. Whats weird is that the multiblock info even states that they would accept 2 hatches however the multiblock does not form.

**Comment by SethaWetha** [comment]
> > Tier skipping was never possible because of adjusted crafting recipes. Also I can still overclock my machines since two 1A Hatches achieve an overclocked power tier. If I were to e.g. change all my 4A hatches to 2x1A hatches nothing would change.
> 
> that's what we wanted to achieve, don't worry.

What is the point of the 4A energy hatches then? Multiblock sharing only? Feels like a waste spending all that time crafting 4A hatches now...

**Comment by TheLastPaladin** [comment]
> What is the point of the 4A energy hatches then? Multiblock sharing only? Feels like a waste spending all that time crafting 4A hatches now...
Its for overclocking as well. I like the chance since it adds a bit of realism (machine speed depending on voltage not on wattage)
However I really dislike the inconsistency. Either all machines can be overclocked the same way or none. Everything else just feels off and isn't fun to play with.

**Comment by SethaWetha** [comment]
If you can still overclock with two normal hatches (2A x 2) then it's not really based on voltage anyways. I don't understand why 2A x 2 is different than 4A.

**Comment by screret** [comment]
to stop recipe tier skips.

**Comment by SethaWetha** [comment]
> to stop recipe tier skips.

Maybe I'm misunderstanding sorry. Couldn't you still just tier skip with two normal hatches then?

**Comment by dimondmine2** [comment]
To my understanding, going up 1 tier with 2 energy hatches in intentional, going up 2 energy ties with more energy hatches/amps is not intentional.

A use case for a 16 amp HV hatch is that if it were wallshared by 4 EBFs I believe it would power all 4 EFBs at EV power

**Comment by SethaWetha** [comment]
> To my understanding, going up 1 tier with 2 energy hatches in intentional, going up 2 energy ties with more energy hatches/amps is not intentional.
> 
> A use case for a 16 amp HV hatch is that if it were wallshared by 4 EBFs I believe it would power all 4 EFBs at EV power
That was my understanding as well, but the response was to "stop recipe tier skips." If you can go up one tier with 2 energy hatches you're not stopping any tier skips with this change.

**Comment by manuelnagler** [comment]
Even if this was closed as "works as intended" I am still waiting for some new issues that came up due to this change to be fixed:
- The Multiblocks that are not taking more than one energy hatch: Is this a bug or a feature? Example: I transitioned to "all multiblocks" but now my assembler can't work on zpm anymore. I was still using the renommated LuV 4A Energy Hatch. I now wanted to make the ZPM Energy Hatch. However, it can only be crafted at ZPM Level. Do I really need the single block assembler now to craft ONE ZPM Energy Hatch? This can't be intended.

- If it stays this way (not allowing even a single tier upgrade) the larger energyhatches should not use transformers as ingredient, since they don't transform the voltage anymore

Fixes:
- Allow multiple energy hatches on every multiblock
- Any multiampere energy hatch should increase the tear by maximum one. Prevents tier skips but reenables the nicer energy input
